docs/index.md

```
@@ -33,431 +33,332 @@ pip install lighter
-    from torchvision.datasets import CIFAR10
-    from torchvision.transforms import ToTensor, Normalize, Compose
+## Quick Comparison
 
+=== "LightningModule"
 
-    class Model(LightningModule):
-        def __init__(self):
-            super().__init__()
-            self.model = resnet18(num_classes=10)
-            self.criterion = CrossEntropyLoss()
+    ```python title="model.py"
```

markdownlint-cli2 (0.18.1)

Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)